container-core/src/main/java/com/yahoo/container/jdisc/state/StateHandler.java

```diff
@@ -274,15 +274,11 @@ private ObjectNode buildJsonForSnapshot(String consumer, MetricSnapshot metricSn
     }
 
     private static List<Tuple> collapseMetrics(MetricSnapshot snapshot, String consumer) {
-        switch (consumer) {
-            case HEALTH_PATH:
-                return collapseHealthMetrics(snapshot);
-            case "all": // deprecated name
-            case METRICS_PATH:
-                return flattenAllMetrics(snapshot);
-            default:
-                throw new IllegalArgumentException("Unknown consumer '" + consumer + "'.");
-        }
+        return switch (consumer) {
+            case HEALTH_PATH -> collapseHealthMetrics(snapshot); // deprecated name
```

Comment:
I think this one got misplaces, should be on "all". And maybe deprecate and add note about removing in Vespa 9?

Reply:
Yes, added a better comment in https://github.com/vespa-engine/vespa/pull/24354